src/redux/missions/missionsSlice.ts

```diff
@@ -44,11 +46,31 @@ export const missionsSlice = createSlice({
     initializeMissions: (state, action: PayloadAction<{ missions: Phase[] }>) => {
       state.isInitialized = true;
       state.data.missions = action.payload.missions.map(convertPhaseToUTC);
+
+      // If no mission was loaded or if the previously selected mission was removed,
+      // automatically select first available mission from the updated list
+      const isSelectedMissionLoaded = state.data.missions.some(
+        (mission) => mission.identifier === state.selectedMissionIdentifier
+      );
+      if (!isSelectedMissionLoaded) {
+        state.selectedMissionIdentifier = action.payload.missions[0].identifier;
+      }
+      return state;
```

Comment:
Now that it is possible to have multipe missions, maybe we want to store them as a map with a key first? that way the finds are easier

Reply:
I hear ya, I'm wondering if it is worth the extra overhead when receiving the data from OpenSpace since we will, for the most part, have 1 mission loaded. If users have loaded multiple ones, I doubt it will be more than 2-3 at most?